ios/preference_bundle_swift/Makefile

```
@@ -0,0 +1,13 @@
+TARGET := iphone:clang:latest:7.0
```

Is Swift intended for iOS 7?

---

Good catch @PoomSmart, I don't think anything below 12.2 is guaranteed to work
```suggestion
TARGET := iphone:clang:latest:12.2
```

---

Yeah, I reckon we can change this to 12.2. Swift still [claims to support](https://swift.org/platform-support/) 7.0+ but on last test (on iPhone 5s 7.1.2) it crashes due to libswiftCore linking a newer libobjc symbol, although this [may be fixed](https://github.com/apple/swift/blob/a55f9cf08cfac1ae5a88d3ec170835810ee8feb9/stdlib/public/runtime/Metadata.cpp#L3132-L3136). Either way, may as well just bump to 12.2 where we also bypass having to worry about the libswift depends.